File: java/dxa-module-flex-layout/src/main/java/com/sdl/dxa/modules/flexlayout/model/ColumnLayoutContainer.java
package com.sdl.dxa.modules.flexlayout.model;

import com.sdl.webapp.common.api.mapping.semantic.annotations.SemanticEntity;
import com.sdl.webapp.common.api.mapping.semantic.annotations.SemanticProperty;
import com.sdl.webapp.common.exceptions.DxaException;

import static com.sdl.webapp.common.api.mapping.semantic.config.SemanticVocabulary.SDL_CORE;

/**
 * Column Layout Container
 *
 * @author nic
 */
@SemanticEntity(entityName = "ColumnLayoutContainer",  vocabulary = SDL_CORE, prefix = "c")
public class ColumnLayoutContainer extends LayoutContainer {

    @SemanticProperty("c:columns")
    private int columns;

    public ColumnLayoutContainer() throws DxaException {
        super();
    }

    public int getColumns() {
        return columns;
    }
}

File: dotnet/SDL.DXA.Modules.FlexLayout/Models/ColumnLayoutContainer.cs
using Sdl.Web.Common.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SDL.DXA.Modules.FlexLayout.Models
{
    /// <summary>
    /// Column Layout Container
    /// </summary>
    [SemanticEntity(EntityName = "ColumnLayoutContainer", Vocab = CoreVocabulary, Prefix = "c")]
    public class ColumnLayoutContainer : LayoutContainer
    {
        [SemanticProperty("c:columns")]
        public int Columns { get; set; }
    }
}